docs/BUILD.md

```diff
@@ -9,39 +9,49 @@ and then read [CUSTOM-BUILD.md](./CUSTOM-BUILD.md).
 
 EVE Makefile automates building and running the following artifacts (all of which are described in gory details below):
 
-* linuxkit/Docker packages
+* OCI images, which are the "building blocks" composed together into the EVE rootfs; these are linuxkit packages, in [pkg/](../pkg/), build via `linuxkit pkg build`
 * EVE rootfs images
 * bootable EVE live images
 * bootable EVE installer images
 
-While linuxkit/Docker packages don't provide any knobs to control the flavor of the build, the rest of the artifacts do.
+![Build Process](./images/build-process.png)
 
-The four main knobs you should be aware of are:
+While the packages don't provide any knobs to control the flavor of the build, the rest of the artifacts do.
+These can be set as variables to `make` targets.
 
-* `ROOTFS_VERSION` - sets the version ID of the EVE image
-* `ZARCH` - sets the architecture (either arm64 or amd64) of the resulting image
-* `HV` - sets the hypervisor flavor of the resulting image (acrn, xen or kvm)
-* `IMAGE_FORMAT` - sets the image format of the resulting bootable image (raw, qcow2 or gcp)
-* `PREEMPT_RT` - (amd64 only) set to '1' applies, enables and builds PREEMPT_RT patches for EVE kernel
+The main knobs that control the flavor of the build are the following. Each of these has reasonably sane
+defaults, when possible.
 
-You can always specify an arbitrary combinations of these knobs to set the desired outcome of the build.
-For example, the following command line will build a Google Compute Platform live image with the default
+| Variable | Description | Default | Examples |
+| --- | --- | --- | --- |
+| `ROOTFS_VERSION` | Version ID of the EVE image | calculated from the git commit | `snapshot`, `1.2.3-abcd567` |
+| `ZARCH` | Hardware architecture of the resulting image | your current platform | `amd64`, `arm64`, `riscv64` |
+| `HV` | Hypervisor flavor of the resulting image | `kvm` (amd64, arm64) or `mini` (riscv64) | `kvm`, `xen`, `acrn`, `mini`, `kubevirt` |
+| `PLATFORM` | Specific platform for which to build | `generic` | `generic`, `rt`, `nvidia`, `imx8mp_pollux` |
+
+Finally, the target determines what type of image you are building, e.g.`live`, `installer`.
+
+As much as possible, incompatible combinations will be caught and prevent a build. For example,
+`PLATFORM=nvidia` is for the ARM-based Jetsons, so `ZARCH=amd64` is incompatible and will be caught.
+
+You can specify an arbitrary combinations of these knobs to set the desired outcome of the build.
+
+For example, the following command line will build a generic qcow2 live image with the default
 hypervisor set to `kvm`, hardware architecture set to `amd64` and the version ID set to `snapshot`:
```

Comment:
the command in the example below uses arm64 for ZARCH, but it says above amd64, so one of these must be corrected....

Reply:
Well, oops. Nice catch. Fixed that.